gnomad_qc/v3/create_release/create_release_sites_ht.py

```diff
@@ -193,49 +220,43 @@ def pre_process_subset_freq(
             ht.freq,
         )
     )
+    ht = ht.select("freq").select_globals("freq_meta")
 
     return ht
 
 
 def main(args):
     hl.init(log="/create_release_ht.log", default_reference="GRCh38")
+    global_freq_path = get_freq(het_nonref_patch=args.het_nonref_patch).path
 
     # The concatenated HT contains all subset frequency annotations, plus the overall cohort frequency annotations,
     # concatenated together in a single freq annotation ('freq')
 
     # Load global frequency Table
     if args.test:
-        global_freq_chr20_ht_path = "gs://gnomad-tmp/gnomad_freq/chr20_test_freq.ht"
+        global_freq_test_ht_path = get_checkpoint_path(
+            f"test_freq{'_patch' if args.het_nonref_patch else ''}"
+        )
 
-        if file_exists(global_freq_chr20_ht_path):
+        if file_exists(global_freq_test_ht_path):
             logger.info(
-                "Loading chr20 global frequency data for testing: %s",
-                global_freq_chr20_ht_path,
-            )
-            global_freq_ht = (
-                hl.read_table(global_freq_chr20_ht_path)
-                .select("freq")
-                .select_globals("freq_meta")
+                "Loading global test frequency data for sites HT testing: %s",
+                global_freq_test_ht_path,
             )
+            global_freq_ht = hl.read_table(global_freq_test_ht_path)
 
-        elif file_exists(get_freq().path):
+        elif file_exists(global_freq_path):
             logger.info(
-                "Loading global frequency data for testing: %s", get_freq().path
-            )
-            global_freq_ht = (
-                hl.read_table(get_freq().path)
-                .select("freq")
-                .select_globals("freq_meta")
+                "Loading global frequency data for testing: %s", global_freq_path
             )
+            global_freq_ht = hl.read_table(global_freq_path)
             global_freq_ht = hl.filter_intervals(
-                global_freq_ht, [hl.parse_locus_interval("chr20:1-1000000")]
+                global_freq_ht, [hl.parse_locus_interval("chr1:1-1000000")]
```

Comment:
basic question, why the change from chr20-chr1? was it too slow to filter the freq ht to chr20 for testing?

Reply:
No, I just used the first 2 partitions in the frequency test code, so this will allow me to still test that those frequencies are being added, otherwise they are all missing. I also didn't fully understand the original use of chr20.